pkg/setting/setting_unified_alerting.go

```diff
@@ -52,8 +52,8 @@ const (
 	// changing this value is discouraged because this could cause existing alert definition
 	// with intervals that are not exactly divided by this number not to be evaluated
 	SchedulerBaseInterval = 10 * time.Second
-	// DefaultAlertForDuration indicates a default interval of for how long a rule should be evaluated to change state from Pending to Alerting
-	DefaultAlertForDuration = 60 * time.Second
+	// DefaultRuleEvaluationInterval indicates a default interval of for how long a rule should be evaluated to change state from Pending to Alerting
+	DefaultRuleEvaluationInterval = SchedulerBaseInterval * time.Duration(6) // == 60 seconds
```

Comment:
```suggestion
	DefaultRuleEvaluationInterval = SchedulerBaseInterval * 6 // == 60 seconds
```

`DefaultRuleEvaluationInterval` is already a time interval, so multiplying it by another interval feels strange. `time.Duration(6)` represents 6 nanoseconds, not seconds, but this ends up working because a Duration is really just an int underneath and `SchedulerBaseInterval` is already in seconds. It'd be a bit clearer to just use the unitless int to begin with.

Reply:
good point!